Meter.h

```diff
@@ -118,6 +132,34 @@ struct Meter_ {
    void* meterData;
 };
 
+/* Used in GraphMeterMode_draw() subroutines only */
+typedef struct GraphDrawContext_ {
+   uint8_t maxItems;
+   bool isPercentChart;
+   size_t nCellsPerValue;
+} GraphDrawContext;
+
+/* Used in GraphMeterMode_computeColors() subroutines only */
+typedef struct GraphColorComputeState_ {
+   double valueSum;
+   unsigned int nCellsPainted;
+   uint8_t nItemsPainted;
+} GraphColorComputeState;
+
+/* Used in GraphMeterMode_computeColors() subroutines only */
+typedef struct GraphColorAdjOffset_ {
+   uint32_t offsetVal; /* "offsetVal" requires at least 22 bits */
+   unsigned int nCells;
+} GraphColorAdjOffset;
+
+/* Used in GraphMeterMode_computeColors() subroutines only */
+typedef struct GraphColorAdjStack_ {
+   double startPoint;
+   double fractionSum;
+   double valueSum;
+   uint8_t nItems;
+} GraphColorAdjStack;
+
```

Comment:
The `GraphColor*` structs are intended to be used in `Meter.c` only. Not sure what you mean by the word "private". Do you wish these definitions in `Meter.c` instead of `Meter.h`?

Reply:
Yes, exactly. `Meter.h` is public interface.

Reply:
Moved to `Meter.c`.